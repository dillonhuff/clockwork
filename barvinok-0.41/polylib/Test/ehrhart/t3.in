2 4
1 1 0 0		# x >= 0
1 -2 1 0		# x <= P

1 3
0 1 -11		P=11

P
instanciated parameter test
---------------------------
0 <= x <= P
P=10
---------------------------
