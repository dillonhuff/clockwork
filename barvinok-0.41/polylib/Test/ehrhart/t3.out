---------------------------------------
Domain:
         P  -11 = 0
          1 >= 0

Vertices:
[   0 ]
[  11 ]/   2 

Ehrhart Polynomial:
( 6 )

